tox.ini

```diff
@@ -7,7 +7,7 @@ env_list = lint, integration
 setenv =
 	version=3.6.4
 	repo=https://github.com/canonical/zookeeper-k8s-operator.git
-	base-name=docker.io/dataplatformoci/zookeeper
+	base-name=ghcr.io/canonical/charmed-zookeeper:3.6.4-22.04_edge
```

Comment:
**niptick** maybe we could use the variable version in the name, just to have a single point and prevent bugs